### PR TITLE
Fix MSVC Libfuzzer coverage reporting

### PR DESCRIPTION
**Pull request by jopletchMSFT**
This PR fixes two issues:
- First, in MSVC compiled binaries both the LLVM _and_ MSVC symbols are
present, but only the MSVC symbols have correct values. For example:

```
0:000> cdb: Reading initial command '.scriptload DumpCountersOld.js ; !dumpcounters "cov" ; q'
JavaScript script successfully loaded from 'DumpCountersOld.js'
[+] not disabling sympath
INFO: Seed: 58715679
INFO: Loaded 1 modules   (3968 inline 8-bit counters): 3968 [00007FF70DB4B000, 00007FF70DB4BF80), # XXX Note
xxx.exe: Running 1 inputs 1 time(s) each.
Running: inp
[+] processing xxx.exe
[+] using LLVM 10 symbols - 0x7ff70db72b00:0x7ff70db72b08 # XXX These are wrong
```

This means the order we search for the coverage symbols is important.

- Secondly, this enables support for MSVC 8bit counter coverage.

## Validation Steps Performed

Running any recent MSVC compiled libfuzzer target should fail to actually collect coverage, instead just returning the 8 null bytes described in the linked issue.

### CODE REVIEW

File: src/agent/script/win64/libfuzzer-coverage/DumpCounters.js [hunk 1]
```diff
@@ -20,15 +20,16 @@ function readU8Array(addr, len) {
...
 // For future research: Other tables of interest in MSVC 16.8
-// _Sancov8bitUsed - __sancov$8bitCountersStart & __sancov$8bitCountersEnd
 // _SancovPcGuardUsed - __sancov$TracePCGuardStart & __sancov$TracePCGuardEnd
 // _SancovPcTableUsed - __sancov$PCTableStart & __sancov$PCTableEnd
 
 function findCounterSymbols(exe) {
     var symbols = [
-        { name: "LLVM 10", start: "__start___sancov_cntrs", end: "__stop___sancov_cntrs" },
         { name: "MSVC 16.8", start: "__sancov$BoolFlagStart", end: "__sancov$BoolFlagEnd" },
+        { name: "MSVC 16.8", start: "__sancov$8bitCountersStart", end: "__sancov$8bitCountersEnd" },
```
Comment (bmc-msft) [inline src/agent/script/win64/libfuzzer-coverage/DumpCounters.js:29]: Can we differentiate between MSVC 17.8 that used BoolFlagStart and 8bitCountersStart by name here?  It  will help in future debugging.

File: src/agent/script/win64/libfuzzer-coverage/DumpCounters.js [hunk 2]
```diff
@@ -20,15 +20,16 @@ function readU8Array(addr, len) {
...
         { name: "MSVC pre-16.8", start: "SancovBitmapStart", end: "SancovBitmapEnd" },
+        // MSVC compiled libfuzzer targets _also_ include the LLVM symbols, so this needs to be checked after MSVC
```
Comment (bmc-msft) [inline src/agent/script/win64/libfuzzer-coverage/DumpCounters.js:31]: :thumbsup:

**Review by bmc-msft — CHANGES_REQUESTED** [review]

File: src/agent/script/win64/libfuzzer-coverage/DumpCounters.js (hunk 1 above)
Comment (jopletchMSFT) [reply src/agent/script/win64/libfuzzer-coverage/DumpCounters.js]: Done -- happy to change the names if you'd prefer them in a different format. Mapped them to the compiler args used to generate them:
```
inline-8bit-counters -> "8bit counters"
inline-bool-flag -> "bool flag"
```

**Review by bmc-msft — APPROVED** [review]

**Comment by bmc-msft** [comment]
Performing integration tests with existing samples.

**Pull request merged**